### ISSUE TITLE
#26415 Broke flower dashboard

### ISSUE THREAD

**Issue by potiuk** [apache/airflow#27402]
### Discussed in https://github.com/apache/airflow/discussions/27401

<div type='discussions-op-text'>

<sup>Originally posted by **Flogue** October 25, 2022</sup>
### Official Helm Chart version

1.7.0 (latest released)

### Apache Airflow version

2.4.1

### Kubernetes Version

1.24.6

### Helm Chart configuration

```
flower:
  enabled: true
```

### Docker Image customisations

None

### What happened

Flower dashboard is unreachable.
"Failed to load resource: net::ERR_CONNECTION_RESET" in browser console

### What you think should happen instead

Dashboard should load.

### How to reproduce

Just enable flower: 

```
helm install airflow-rl apache-airflow/airflow --namespace airflow-np --set flower.enables=true
kubectl port-forward svc/airflow-rl-flower 5555:5555 --namespace airflow-np
```

### Anything else

A quick fix for this is:

```
config:
  celery:
    flower_url_prefix: ''
```

Basically, the new default value '/' makes it so the scripts and links read:
`<script src="//static/js/jquery....`
where it should be:
`<script src="/static/js/jquery....`

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)
</div>

**Comment by potiuk** [comment]
converted to discussion by mistake. Marked as good first issue - flower is not a high priority but hopefully someone will fix it (howeve note @Flogue that fastest way to  get it implemented is if you implement it - this is Open Source and you are most welcome to provide PRs and you can become one of the > 2200 contributors.

**Comment by rajat-tomar** [comment]
@potiuk can you please assign this to me? I would like to work on it.

**Comment by potiuk** [comment]
Assigned :)

**Comment by amoghrajesh** [comment]
Hey, looks like this issue went stale. I'd like to give it a try and fix it. Self assignign
